apps/wallet/src/ui/app/shared/PageContainer.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type ReactNode } from 'react';
+
+interface Props {
+    children: ReactNode;
+}
+
+export function PageContainer({ children }: Props) {
+    return (
+        <div className="flex flex-col h-full overflow-x-visible">
+            {children}
+        </div>
+    );
+}
+
+export function PageContent({ children }: Props) {
```

Comment:
Another way to do that (that I generally prefer) is to use position sticky and make the header sticky.